config_sample.toml

```diff
@@ -37,15 +38,15 @@
 ## Valid values:
 ## auto - automatically select a suitable homeserver
 ## A URL pointing to a Raiden matrix homeserver
-# matrix-server = "auto"
+#matrix-server = "auto"
 
-# log-file =
-# log-json = false
+#log-file =
+#log-json = false
 
-# rpc = true
-# rpccorsdomain = "http://localhost:*/*"
-# api-address = "127.0.0.1:5001"
-# web-ui = true
+#rpc = true
+#rpccorsdomain = "http://localhost:*/*"
+#api-address = "127.0.0.1:5001"
+#web-ui = true
 
-# [log-config]
-# "" = "INFO"
+#[log-config]
+#"" = "DEBUG"
```

Comment:
Do we really want to enable debug by default?

Reply:
Ah, no, that's a leftover from testing. Good catch :)